src/CoreFoundation/CFMutableString.cs

```diff
@@ -102,7 +104,7 @@ public bool Transform (ref CFRange range, NSString transform, bool reverse)
 
 		public bool Transform (ref CFRange range, string transform, bool reverse)
 		{
-			var t = NSString.CreateNative (transform);
+			var t = CreateNative (transform);
```

Comment:
Context, NSString.CreateNative (...) is obsolete and the recommendation was to use CFString.CreateNative (...)